crates/configure-holochain/src/lib.rs

```diff
@@ -118,3 +122,17 @@ async fn install_ui(happ: &Happ, config: &Config) -> Result<()> {
     }
     Ok(())
 }
+
+pub async fn update_host_jurisdiction_if_changed(config: &Config) -> Result<()> {
+    if std::env::var("IS_INTEGRATION_TEST")? == "TRUE" {
+        // set in ../tests/integration.rs and ../../holo_happ_manager/tests/integration.ts
+        return Ok(());
+    }
```

Comment:
obv this isn't pretty. The alternative would be to create a mock hpos_holochain_api endpoint. lmk if you think it's worth it